### ISSUE TITLE
Cannot run as standalone binary even after "setcap  CAP_NET_BIND_SERVICE"

### ISSUE THREAD

**Issue by SLCCL** [0xERR0R/blocky#366]
Hello there, 
I'm wondering why i can not run main.go even i add the cap top the execute file.(as pic)

Any help would be appreciated.

[Pic](https://ibb.co/TgQVhVR)

**Comment by kwitsch** [comment]
It seems like you send the bind permission to the go file instead of the executable.
If you directly execute the go project changing the port to some unprivileged one would be best practice.
Other solutions would be to give go itself the permission or execute it as root user(I strongly discourage this options).

If you aren't interested in the sources, you should download a precompiled executable(or compile it yourself) and set the permission to it.

**Comment by 0xERR0R** [comment]
Agreed with @kwitsch, it looks like you run blocky with "go run ...". If you want to build blocky from source, please run `make build`, than you can find the binary in ./bin directory

**Comment by SLCCL** [comment]
Thank you guys so much! 
I think it's time for me to finish the CS50 course.
Thanks again and you guys and the blocky are amazing.
